Makefile

```diff
@@ -99,17 +99,20 @@ remove-agents:
 	docker-compose -f oms-monitor-agents/docker/docker-compose.yml down
 
 # Backups
+backup:
+	/opt/bu-settings/dump.sh postgres-core postgres-events postgres-statutory postgres-discounts
+
 backup_core:
-	./helper.sh --execute postgres-core -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/core' --inserts > core.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-core
 
 backup_events:
-	./helper.sh --execute postgres-events -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/events' --inserts > events.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-events
 
 backup_statutory:
-	./helper.sh --execute postgres-statutory -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/statutory' --inserts > statutory.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-statutory
 
 backup_discounts:
-	./helper.sh --execute postgres-discounts -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/discounts' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-discounts
```

Comment:
why the absolute path and not relative?

Reply:
explicit is better than implicit

Reply:
but it won't work if we'll have different folder structure, it's not standardized yet. and it won't work locally (for example I have the docker setup at `/Users/sergeypeshkov/oms-docker`)

Reply:
the backup scripts is not depending on the location of myaegee

Reply:
what do you think of moving it under scripts/ directory, then invoking it as scripts/dump.sh with relative name?

Reply:
as i mentioned in the other comment, probably ok, I'll give a thought (have to check the ansible flow)